### DISCUSSION TITLE
version downgrade

### DISCUSSION THREAD

**Discussion by sreedeviMullaguri** [jbangdev/jbang#1530] [Q&A]
how to downgrade jbang to specific version

**Comment by maxandersen** [comment]
`rm -rf ~/.jbang/bin/jbang* && curl -Ls https://sh.jbang.dev | JBANG_DOWNLOAD_VERSION=0.92.0 bash -s - app setup`

just set JBANG_DOWNLOAD_VERSION to the version you are after.

**Reply by sreedeviMullaguri** [reply]
rm -rf ~/.jbang/bin/jbang* && curl -Ls https://sh.jbang.dev | JBANG_DOWNLOAD_VERSION=0.95.0 bash -s - app setup
Downloading JBang 0.95.0...
Error downloading JBang from https://github.com/jbangdev/jbang/releases/download/v0.95.0/jbang.tar

**Reply by sreedeviMullaguri** [reply]
im getting error like this...can you please look into it

**Reply by maxandersen** [reply]
If You are getting such errors My guess is you are behind a proxy or similar that prevents access to GitHub.com directly ?
